src/components/forms/AddServiceTypeForm.js

```diff
@@ -93,7 +80,7 @@ function AddServiceTypeForm({
             label="Program"
             rules={[
               {
-                required: true,
+                // required: true
```

Comment:
Thanks for the additional information about why you had to comment this out. Sounds like there may be a mismatch in naming between the old db schema and the new one that was pushed in Labs 37.